lib/slave/src/AbstractESC.cc

```diff
@@ -365,8 +365,9 @@ namespace kickcat
 
     void AbstractESC::set_state_on_error(State state, StatusCode error_code)
     {
+        write(reg::AL_STATUS_CODE, &error_code, sizeof(error_code));
```

Comment:
This looks like a conflict during the rebase

Reply:
Indeed. Should be fixed now.